src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -35,18 +35,34 @@ method groovy.lang.Closure setDelegate java.lang.Object
 method groovy.lang.Closure setResolveStrategy int
 method groovy.lang.GString plus java.lang.String
 method groovy.lang.Script getBinding
+method groovy.util.Node get java.lang.String
+method groovy.util.NodeList getAt java.lang.String
+
+#There are other issues whith these classes than the whitelisting, but should at least be here
+new groovy.util.XmlParser
+new groovy.util.XmlSlurper
+method groovy.util.XmlParser parseText java.lang.String
+method groovy.util.XmlSlurper parseText java.lang.String
+new groovy.xml.MarkupBuilder java.io.Writer
```

Comment:
See #39; I would rather this be pulled out and considered separately with end-to-end tests.